Source/WebCore/css/DeprecatedCSSOMPrimitiveValue.cpp

```diff
@@ -130,7 +130,7 @@ ExceptionOr<Ref<DeprecatedCSSOMCounter>> DeprecatedCSSOMPrimitiveValue::getCount
     if (!m_value->isCounter())
         return Exception { InvalidAccessError };
     auto& value = downcast<CSSCounterValue>(m_value.get());
-    return DeprecatedCSSOMCounter::create(value.identifier(), value.separator(), value.listStyle());
+    return DeprecatedCSSOMCounter::create(value.identifier(), value.separator(), value.listStyleValue()->valueID());
```

Comment:
Is this always correct? I think valueID won't always exist or be valid, if the type is string or custom ident.

Reply:
using `isValueID()` is fine because it returns `CSSValueInvalid` when the CSSValue is not a ValueID, however I need to transform the lambda I created for converting to string into a function so I can re-use it here.